src/styles/leaderboard.css

```
@@ -302,4 +313,4 @@
   .leaderboard-table td:nth-of-type(8):before {
     content: "GAS";
   }
-}
+}
```

In general, ending with a newline

```suggestion
}

```